python/pylibcudf/pylibcudf/libcudf/utilities/traits.pxd

```diff
@@ -1,27 +1,27 @@
-# Copyright (c) 2024, NVIDIA CORPORATION.
+# Copyright (c) 2024-2025, NVIDIA CORPORATION.
 from libcpp cimport bool
 from libcpp.vector cimport vector
 from pylibcudf.exception_handler cimport libcudf_exception_handler
 from pylibcudf.libcudf.types cimport data_type
 
 
 cdef extern from "cudf/utilities/traits.hpp" namespace "cudf" nogil:
-    cdef bool is_relationally_comparable(data_type)
-    cdef bool is_equality_comparable(data_type)
-    cdef bool is_numeric(data_type)
-    cdef bool is_numeric_not_bool(data_type)
-    cdef bool is_index_type(data_type)
-    cdef bool is_unsigned(data_type)
-    cdef bool is_integral(data_type)
-    cdef bool is_integral_not_bool(data_type)
-    cdef bool is_floating_point(data_type)
-    cdef bool is_boolean(data_type)
-    cdef bool is_timestamp(data_type)
-    cdef bool is_fixed_point(data_type)
-    cdef bool is_duration(data_type)
-    cdef bool is_chrono(data_type)
-    cdef bool is_dictionary(data_type)
-    cdef bool is_fixed_width(data_type)
-    cdef bool is_compound(data_type)
-    cdef bool is_nested(data_type)
-    cdef bool is_bit_castable(data_type, data_type)
+    cdef bool is_relationally_comparable(data_type) except +libcudf_exception_handler
+    cdef bool is_equality_comparable(data_type) except +libcudf_exception_handler
+    cdef bool is_numeric(data_type) except +libcudf_exception_handler
+    cdef bool is_numeric_not_bool(data_type) except +libcudf_exception_handler
```

Comment:
I still get the same error even after adding the `libcudf_exception_handler`
```python
In [1]: import pylibcudf as plc
   ...: plc.traits.is_numeric_not_bool(plc.DataType(plc.TypeId.EMPTY))
terminate called after throwing an instance of 'cudf::logic_error'
  what():  CUDF failure at:/home/coder/cudf/cpp/include/cudf/utilities/type_dispatcher.hpp:567: Invalid type_id.
Aborted (core dumped)
```

Reply:
Try doing a clean your pylibcudf (not cudf) build (i.e. make sure the pylibcudf build directory etc are completely wiped).

Reply:
Thanks! A clean build worked